{{cookiecutter.project_name}}/Data/sample_treatment/simple_repeat.py

```diff
@@ -0,0 +1,43 @@
+#a script to run several replicates of several treatments locally
+#You should create a directory for your result files and run this script from within that directory
+
+seeds = range(21, 41)
+#start_mois = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 30, 40, 60, 80, 100]
+#start_mois = [1]
+#slrs = [15]
+#verts = [0.1, 0.3, 0.7]
+verts = [0.3]
+#sym_ints = [-1, -0.9, -0.8, -0.7, -0.6, -0.5, -0.4, -0.3, -0.2, -0.1, 0]
+#h_mut_rate = [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0]
+h_mut_rate = [0.1, 0.5, 1.0]
+
+import subprocess
+
+def cmd(command):
+    '''This wait causes all executions to run in sieries.                          
+    For parralelization, remove .wait() and instead delay the                      
+    R script calls unitl all neccesary data is created.'''
+    return subprocess.Popen(command, shell=True).wait()
+
+def silent_cmd(command):
+    '''This wait causes all executions to run in sieries.                          
+    For parralelization, remove .wait() and instead delay the                      
+    R script calls unitl all neccesary data is created.'''
+    return subprocess.Popen(command, shell=True, stdout=subprocess.PIPE).wait()
+
+print("Copying SymSettings.cfg and executable to current directory")
+cmd("cp ../../SymSettings.cfg .")
```

Comment:
I think I've decided it's best not to copy the SymSettings and symbulation executable and instead assume it's in the folder already